hathor/transaction/resources/push_tx.py

```diff
@@ -98,7 +105,7 @@ def handle_push_tx(self, params: Dict[str, Any]) -> Dict[str, Any]:
                     'message': 'Transaction has an output script that is too big.',
                     'can_force': False,
                 }
-            if not self.allow_non_standard_script:
+            if not self.allow_non_standard_script and not is_nft_creation:
```

Comment:
Should we add NFT Creation as part of the standard scripts? I mean, change the `txout.is_standard_script()`? In fact, maybe we should move it to `tx.is_standard()` which will call `txout.is_standard_script()` by default.

Another approach would be to add a `is_standard_tx(...)` in hathorlib and use it here.

Reply:
I guess the best solution for this would be to add the method in hathorlib, in order to use it on the full node and tx mining service. Should I create a new PR in the lib?

Reply:
I feel this code is still outdated, right?

Reply:
Yes, can you see again, please?